tests/checks/preflight.go

```diff
@@ -81,15 +118,7 @@ func (d *DiscoveryError) Error() string {
 	return fmt.Sprintf("%v", d.errors)
 }
 
-func VerifyClusterExpectations() error {
-	expected := &ClusterProfile{}
-
-	// XXX this flags needs to go
-	// it makes dual stack the default assumption if no profile file is provided
-	if len(flags.ClusterProfilePath) == 0 {
-		deprecatedIsDualStack := !flags.SkipDualStackTests
-		expected.DualNetworkStack = &deprecatedIsDualStack
-	}
+func (c *clusterIntrospector) verifyClusterExpectations(expected ClusterProfile, DiscoveredClusterProfile ClusterProfile) error {
 
 	if len(flags.ClusterProfilePath) > 0 {
```

Comment:
Shouldn't it be removed since you already open the file in the `loadExpectedClusterProfile` member function?